eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServer.java

```diff
@@ -118,10 +118,8 @@ public QueryResponse getVariantById(@PathVariable("variantId") String variantId,
         }
 
 
-        QueryResult<VariantEntity> queryResult = new QueryResult<>();
-        queryResult.setNumResults(variantEntities.size());
+        QueryResult<VariantEntity> queryResult = Utils.buildQueryResult(variantEntities);
         queryResult.setNumTotalResults(numTotalResults);
```

Comment:
To save having to do this in some of the classes, the Utils method buildQueryResult could be overloaded. e.g.:

    public static <T> QueryResult<T> buildQueryResult(List<T> results) {
        return buildQueryResult(results, results.size());
    }

and

    public static <T> QueryResult<T> buildQueryResult(List<T> results, int numTotalResults) {
        QueryResult<T> queryResult = new QueryResult<>();
        queryResult.setResult(results);
        queryResult.setNumResults(results.size());
        queryResult.setNumTotalResults(numTotalResults);
        return queryResult;
    }